README.md

```diff
@@ -28,9 +28,9 @@ Creates a custom permission set SampleDigitalLendingClone.
 Assigns this and several other permission sets to the user.
 </details>
 
-1. Navigate to `Document Generation`'s `General Settings` from Setup (`Feature Settings` &rarr; `Document Generation` &rarr; `General Settings`), and enable `Design Document Templates in Salesforce`.
+2. Navigate to `Document Generation`'s `General Settings` from Setup (`Feature Settings` &rarr; `Document Generation` &rarr; `General Settings`), and enable `Design Document Templates in Salesforce`.
```

Comment:
@SiemelNaran Based on what is mentioned [here in slack](https://salesforce-internal.slack.com/archives/C02C3FZHG9M/p1719950092679499?thread_ts=1719511191.238019&cid=C02C3FZHG9M) Should we remove this point #2 as well?

Reply:
cc @mitsenthil